keyboards/handwired/dc/mc/001/keymaps/default/keymap.c

```diff
@@ -0,0 +1,31 @@
+/* Copyright 2021 Daniel Cormier
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#include QMK_KEYBOARD_H
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+    LAYOUT(KC__MUTE, KC_MEDIA_STOP, KC_MEDIA_PREV_TRACK, KC_MEDIA_NEXT_TRACK, KC_MEDIA_PLAY_PAUSE),
```

Comment:
```suggestion
    LAYOUT(KC_MUTE, KC_MEDIA_STOP, KC_MEDIA_PREV_TRACK, KC_MEDIA_NEXT_TRACK, KC_MEDIA_PLAY_PAUSE),
```

Reply:
Can you explain a little? The docs aren't the most helpful in this area. [`Basic Keycodes` → `Commands`](https://docs.qmk.fm/#/keycodes_basic?id=commands) has `KC__MUTE`, while [`Basic Keycodes` → `Media Keys`](https://docs.qmk.fm/#/keycodes_basic?id=media-keys) has `KC_AUDIO_MUTE`/`KC_MUTE`, but there's no explanation of what the difference is.

Reply:
https://docs.qmk.fm/#/keycodes?id=basic-keycodes

Reply:
Aha! [The beta docs have another page](https://beta.docs.qmk.fm/using-qmk/simple-keycodes/keycodes) that indicates OS support. `KC_AUDIO_MUTE`/`KC_MUTE` shows it's supported on Windows/macOS/Linux, while `KC__MUTE` only shows support on macOS/Linux. Since I tested on macOC, that explains why it worked fine for me.

Updated.